package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "type-safe-errors",
-  "version": "0.1.9",
+  "version": "0.2.0",
```

Comment:
this suggests you are using `0.x` minor versions for breaking changes, which is fine, but you might want to recommend using `~0.2.0` (instead of the default `^0.2.0`) in the user's `package.json` so they don't update to `0.3.x` automatically expecting no breaking changes

Reply:
this is no breadking change - you can still use old style of error. This change is more like update of best practices. 
I use minor version for features (which current thins is really not ;) )

Reply:
I going to merge it after https://github.com/wiktor-obrebski/type-safe-errors/pull/11 , and set the version to `0.2.1`